code/index.html

```diff
@@ -3,28 +3,143 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Lost & Found</title>
+    <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <!-- HEADER -->
+    <header class="header-wrapper">
+        <div class="header-l">
+          <h1 class="pap">PAP.</h1>
+        </div>
 
-    <!-- Signup form -->
+        <div class="header-r">
+          <div class="hamburger-fake"> 
+            <h2 class="hamburger">
+              =
+            </h2>
+          </div>  
+          <button class="contact">Contact</button>
+        </div>
+    </header>
+
+    <div class="hero-video">
+    <video class="hero-video1" autoplay loop muted controlslist="nodownload">
+      <source src="./videos/pap_hands.mp4" type="video/mp4">
+      <source src="./videos/pap hands-1.webm" type="video/webm">
+
+      <source src="./videos/pap_hands.mp4" type="video/mp4" media="(max-width: 991px)">
+      <source src="./videos/pap hands-1.webm" type="video/webm" media="(max-width: 991px)">
+
+      <source src="./videos/pap_hands.mp4" type="video/mp4" media="(max-width: 767px)">
+      <source src="./videos/pap hands-1.webm" type="video/webm" media="(max-width: 767px)">
+      <p>This browser does not support video ¯\_(ツ)_/¯</p>
+    </video>
 
     <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+      <h2 id="rituals" class="header2">LOST & FOUND? - click here</h2>
+      <div id="big-container" class="big-container">
+        <div id="section-1" class="question">
+          <h3 id="accordion-header-1" class="accordion-header">YOU ARE PRESENT AT THE CEREMONY OF?          
+          </h3>
+      
+        <div id="accordion-content-1" class="answer">  <!--THIS MIGHT BE REDUNDANT to have a <p id="" inside of this div, but keep for now-->
+          <p class="accordion-text">The vulva lost and found. <br> All text in this section are 
```

Comment:
We should really avoid using `<br />` tags, they are deprecated - instead, you can do something like this :

```
<p class="accordion-text">
  <div>The vulva lost and found. </div>
  <div>All text in this section are extracts from Monique Wittig's book <i>The Lesbian body</i>.</div>
</p>
```